firebase-database/src/main/java/com/google/firebase/database/tubesock/WebSocket.java

```diff
@@ -14,7 +14,6 @@
 
 package com.google.firebase.database.tubesock;
 
-import android.net.SSLCertificateSocketFactory;
```

Comment:
Intentional?

Reply:
Avoiding deprecation warnings, which are configured as hard errors in these projects